### PR TITLE
IDMP-646 add CTCAE 

### PR DESCRIPTION
**Pull request by tw-osthus**
Description:
add CTCAE as vocabulary for adverse events and CTCAE grades for use in classification of the severity of adverse events

this is used in NA jurisdiction for undesirable effects in the clinical particulars

### CODE REVIEW

**Comment by mereolog** [comment]
@tw-osthus The new ontology will not be a subject of hygiene and other tests until it is added to https://github.com/edmcouncil/idmp/blob/master/AboutIDMPProd-ReferenceIndividuals.rdf.

Also if we want it to be accessible through https://spec.edmcouncil.org/idmp/ontology, we need to add it to https://github.com/edmcouncil/idmp/blob/master/ISO/MetadataISO.rdf as well.

**Comment by ElisaKendall** [comment]
> @tw-osthus The new ontology will not be a subject of hygiene and other tests until it is added to https://github.com/edmcouncil/idmp/blob/master/AboutIDMPProd-ReferenceIndividuals.rdf.
> 
> Also if we want it to be accessible through https://spec.edmcouncil.org/idmp/ontology, we need to add it to https://github.com/edmcouncil/idmp/blob/master/ISO/MetadataISO.rdf as well.

Yup - I'll help him get that done, no problem. I said that I would work on it when we met yesterday.

File: ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+				</owl:oneOf>
+			</owl:Class>
+		</owl:equivalentClass>
+		<skos:definition>classifier for the severity of an adverse event (AE)</skos:definition>
+		<cmns-av:abbreviation>CTCAE grade term</cmns-av:abbreviation>
+		<cmns-av:adoptedFrom>https://ctep.cancer.gov/protocoldevelopment/electronic_applications/docs/CTCAE_v5_Quick_Reference_5x7.pdf</cmns-av:adoptedFrom>
+	</owl:Class>
+	
+	<owl:NamedIndividual rdf:about="&idmp-ctcae;CommonTerminologyCriteraForAdverseEventsClassificationScheme">
+		<rdfs:type rdf:resource="https://spec.pistoiaalliance.org/idmp/ontology/ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents/cmns-cls;ClassificationScheme"/>
```
Comment (mereolog) [inline ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf:143]: This IRI looks wrong. Did you mean https://www.omg.org/spec/Commons/Classifiers/ClassificationScheme?

File: ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+		<rdfs:type rdf:resource="&mvf;Vocabulary"/>
+		<skos:definition>The NCI Common Terminology Criteria for Adverse Events is a descriptive terminology which can be utilized for Adverse Event (AE) reporting. A grading (severity) scale is provided for each AE term.</skos:definition>
+		<skos:note>Entries in the vocabulary denoting adverse event types are registered in MedDRA.</skos:note>
+		<cmns-av:abbreviation>CTCAE vocabulary</cmns-av:abbreviation>
+		<cmns-av:synonym>CTCAE terminology</cmns-av:synonym>
+		<cmns-av:synonym>Common Terminology Criteria for Adverse Events terminology</cmns-av:synonym>
+		<cmns-ra:isManagedBy rdf:resource="&idmp-nara;NationalCancerInstitute"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="https://spec.pistoiaalliance.org/idmp/ontology/ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents/idmp-ctcae;CTCAE-Grade1">
```
Comment (mereolog) [inline ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf:158]: This IRI looks suspicious - did you mean `&idmp-ctcae;CTCAE-Grade1`?

File: ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+		<rdf:type rdf:resource="&idmp-ctcae;CTCAE-GradeTerm"/>
+		<rdfs:label>Grade 1</rdfs:label>
+		<skos:definition>Mild; asymptomatic or mild symptoms; clinical or diagnostic observations only; intervention not indicated.</skos:definition>
+		<idmp-dtp:hasDisplayName>Grade 1</idmp-dtp:hasDisplayName>
+		<cmns-av:directSource>https://ctep.cancer.gov/protocoldevelopment/electronic_applications/docs/CTCAE_v5_Quick_Reference_5x7.pdf</cmns-av:directSource>
+		<cmns-txt:hasTextValue>Grade 1</cmns-txt:hasTextValue>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="https://spec.pistoiaalliance.org/idmp/ontology/ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents/idmp-ctcae;CTCAE-Grade2">
```
Comment (mereolog) [inline ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf:167]: See my comment to line 158 above.

File: ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+		<rdf:type rdf:resource="&idmp-ctcae;CTCAE-GradeTerm"/>
+		<rdfs:label>Grade 2</rdfs:label>
+		<skos:definition>Moderate; minimal, local or noninvasive intervention indicated; limiting ageappropriate instrumental ADL.</skos:definition>
+		<idmp-dtp:hasDisplayName>Grade 2</idmp-dtp:hasDisplayName>
+		<cmns-av:directSource>https://ctep.cancer.gov/protocoldevelopment/electronic_applications/docs/CTCAE_v5_Quick_Reference_5x7.pdf</cmns-av:directSource>
+		<cmns-txt:hasTextValue>Grade 2</cmns-txt:hasTextValue>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="https://spec.pistoiaalliance.org/idmp/ontology/ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents/idmp-ctcae;CTCAE-Grade3">
```
Comment (mereolog) [inline ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf:176]: See my comment to line 158 above.

File: ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+		<rdf:type rdf:resource="&idmp-ctcae;CTCAE-GradeTerm"/>
+		<rdfs:label>Grade 3</rdfs:label>
+		<skos:definition>Severe or medically significant but not immediately life-threatening; hospitalization or prolongation of hospitalization indicated; disabling; limiting self care ADL.</skos:definition>
+		<idmp-dtp:hasDisplayName>Grade 3</idmp-dtp:hasDisplayName>
+		<cmns-av:directSource>https://ctep.cancer.gov/protocoldevelopment/electronic_applications/docs/CTCAE_v5_Quick_Reference_5x7.pdf</cmns-av:directSource>
+		<cmns-txt:hasTextValue>Grade 3</cmns-txt:hasTextValue>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="https://spec.pistoiaalliance.org/idmp/ontology/ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents/idmp-ctcae;CTCAE-Grade4">
```
Comment (mereolog) [inline ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf:185]: See my comment to line 158 above.

File: ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+		<rdf:type rdf:resource="&idmp-ctcae;CTCAE-GradeTerm"/>
+		<rdfs:label>Grade 4</rdfs:label>
+		<skos:definition>Life-threatening consequences; urgent intervention indicated.</skos:definition>
+		<idmp-dtp:hasDisplayName>Grade 4</idmp-dtp:hasDisplayName>
+		<cmns-av:directSource>https://ctep.cancer.gov/protocoldevelopment/electronic_applications/docs/CTCAE_v5_Quick_Reference_5x7.pdf</cmns-av:directSource>
+		<cmns-txt:hasTextValue>Grade 4</cmns-txt:hasTextValue>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="https://spec.pistoiaalliance.org/idmp/ontology/ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents/idmp-ctcae;CTCAE-Grade5">
```
Comment (mereolog) [inline ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf:194]: See my comment to line 158 above.

**Review by mereolog — CHANGES_REQUESTED** [review]

File: ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf (hunk 1 above)
Comment (tw-osthus) [reply ISO/NorthAmericanJurisdiction/CommonTerminologyCriteriaForAdverseEvents.rdf:143]: yes, fixed.

**Review by ElisaKendall — APPROVED** [review]
This is fine now with some additional changes to fix spelling issues, definition formats, eliminate an unnecessary catalog file, use mvf-trm;Term and mvf;VocabularyEntry where needed, and the like.

**Review by mereolog — CHANGES_REQUESTED** [review]
Running test              : [If we refer to something, it should have a type]
	ERROR:https://spec.pistoiaalliance.org/idmp/ontology/ISO/NorthAmericanJurisdiction//NorthAmericanRegistrationAuthorities/NationalCancerInstitute is referenced by isManagedBy but has no type.

**Comment by ElisaKendall** [comment]
> Running test : [If we refer to something, it should have a type] ERROR:https://spec.pistoiaalliance.org/idmp/ontology/ISO/NorthAmericanJurisdiction//NorthAmericanRegistrationAuthorities/NationalCancerInstitute is referenced by isManagedBy but has no type.

I'm looking for that now - once this passes, please approve

**Review by ElisaKendall — APPROVED** [review]

**Review by mereolog — APPROVED** [review]

**Pull request merged**